mlrun/datastore/sources.py

```diff
@@ -412,6 +413,51 @@ def to_dataframe(
             **reader_args,
         )
 
+    def _get_spark_additional_filters(self):
+        if not self.additional_filters:
+            return None
+        none_values = [None, np.nan, np.NaN]
+        additional_filters = deepcopy(self.additional_filters)
+        is_first = True
+        spark_filter = ""
+        for filter_tuple in additional_filters:
+            if not filter_tuple:
+                continue
+            if not is_first:
```

Comment:
```suggestion
            if spark_filter:
```
No need for the extra variable.

Reply:
nice!